gatling/simulations/navitia/distributed.scala

```diff
@@ -0,0 +1,32 @@
+package navitia
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+
+object Base {
+
+  val httpConf = http
+        .baseURL("http://localhost:5000/v1/coverage") //jormun
+        .acceptEncodingHeader("gzip, deflate")
+
+  def query(scn: String) = {
+    val requests = csv("journeys_fr-idf.csv").circular
+      exec().feed(requests).exec(
+        http("${type}")
+          .get("/${coverage}/journeys?${other_args}")
+          .queryParam("from", "${from}")
+          .queryParam("to", "${to}")
+          .queryParam("datetime", "${datetime}")
+          .queryParam("datetime_represents", "${datetime_represents}")
+          .queryParam("_override_scenario", scn)
+          .check(status.is(200)))
+  }
+}
+
+class Distributed extends Simulation {
+    setUp(scenario("distributed").exec(Base.query("distributed")).inject(constantUsersPerSec(1) during(300) randomized).protocols(Base.httpConf))
```

Comment:
OK, I guess that you used this: https://gatling.io/docs/current/general/simulation_setup/#simulation-setup-throttling
But I'm pretty sure I miss something here: You're injecting 300(1 per second) users during 300s, at random moments, is that it?
Then is that random part repeatable and the same for both simulations?

Reply:
that's random without being random, it doesn't inject a user at the start of each second, so the charge isn't perfectly uniform.
It's not required for our use case but I like to add it when doing real load testing.